model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -195,6 +217,7 @@ class ModelReplicationServer(
         branch: String,
         lastKnown: String?,
     ) {
+        checkPermission("repository", repository, "branch", branch, "pull")
```

Comment:
`postRepositoryObjectsGetAll(repository: String)` is not secured.

Reply:
Added a check.